### ISSUE TITLE
Use the "not" notation for switch positions

### ISSUE THREAD

**Issue by phaedra-tm** [FrSkyRC/ETHOS-Feedback-Community#1510]
Currently using 1.1.1 EU Firmware on X20S
In the current version, we can select a specific switch position as a condition, for instance, SD up.
What is missing, is the possibility to specify "NOT SD up", meaning either SD mid or SD down.

**Comment by MIKE-DAILEY** [comment]
Whenever you see the small square in the upper left corner of a cell it indicates more options with a long press, e.g. invert.

**Comment by phaedra-tm** [comment]
Ok, thanks for the reply.
I did try pressing that square multiple times  before, but it kept giving me the list of switch positions.  I do have thin female fingers, but they appear to be too thick for this sort of thing.
After your tip, I tried pushing it with my fingernail, because it is so incredibly tiny in the 6-widget telemetry screen, and low and behold, I could see these options.
I still believe it makes more sense to put these options in the list of switch positions, like in OTX, because as it is now, this is not only very hard to find, but also very hard to use.  Or maybe make that tiny square a bit larger?
But again, I really appreciate you taking the time to give me this info, which is probably in the manual (which I consulted, but I must have overlooked this completely).
